pyrdfj2/j2_functions.py

```diff
@@ -0,0 +1,202 @@
+import re
+from collections.abc import Iterable
+from datetime import date, datetime
+
+from dateutil import parser
+from uritemplate import URITemplate
+
+
+class Functions:
+    _cache = dict()
+
+    @staticmethod
+    def all():
+        return {
+            # TODO -- remove this in 0.2.0 -- deprecated use, now use
+            #   `| ttl(...)` instead
+            "ttl_fmt": turtle_format,
+            "uritexpand": uritexpand,
+            "regexreplace": regexreplace,
+            "map": map_build,
+            # TODO -- remove this in 0.2.0 -- depracted use, no longer needed
+            #    since xmlasdict
+            "unparse": xml_unparse,
+        }
+
+
+class Filters:
+    @staticmethod
+    def all():
+        return {
+            "ttl": turtle_format,  # new way of using this formatting
```

Comment:
and since this is also more about ^^xsd type formatting then about turtle (or sparql)   we might want to fix this naming too?

(less important though, and not finding a good moniker)

Reply:
`xml_schema_formatting`?